components/gauge/include/bq27441_defs.h

```diff
@@ -0,0 +1,117 @@
+#ifndef _BQ27441_DEFS_H_
+#define _BQ27441_DEFS_H_
+
+/**
+ * @file bq27441_defs.h
+ * @brief Definitions for the BQ27441 fuel gauge.
+ */
+
+#define BQ27441_DATA_SIZE 3 * sizeof(uint16_t) /**< Size of the buffer to hold capacity, SoC, and voltage data*/
+
+/**
+ * @brief Sensor interface functions for BQ27441.
+ */
+extern sensor_interface_t bq27441_interface;
```

Comment:
Since BQ27441 is not a sensor, consider renaming sensor_interface_t and bq27441_interface to avoid confusion, e.g., battery_interface_t or power_management_interface_t

Reply:
maybe its not a sensor in a obvious way, but it 'measures' battery voltage and capacity. Also its i2c device.